index.bs

```diff
@@ -1258,19 +1278,19 @@ During the above process, the user agent SHOULD show some UI to the user to guid
 authorizing an authenticator with which to complete the operation.
 </div>
 
-### Store an existing credential - PublicKeyCredential's `[[Store]](credential)` method ### {#storeCredential}
+### Store an existing credential - PublicKeyCredential's `[[Store]](credential, sameOriginWithAncestors)` method ### {#storeCredential}
 
-<div link-for-hint="PublicKeyCredential/[[Store]](credential)">
+<div link-for-hint="PublicKeyCredential/[[Store]](credential, sameOriginWithAncestors)">
 
-The <dfn for="PublicKeyCredential" method>\[[Store]](credential)</dfn> method is not supported
+The <dfn for="PublicKeyCredential" method>\[[Store]](credential, sameOriginWithAncestors)</dfn> method is not supported
 for Web Authentication's {{PublicKeyCredential}} type, so it always returns an error.
 
 Note: This algorithm is synchronous; the {{Promise}} resolution/rejection is handled by
 {{CredentialsContainer/store()|navigator.credentials.store()}}.
 
 This method accepts a single argument:
```

Comment:
This is technically no longer true, although it seems a little weird to specify the parameters at all when they are in effect ignored.